src/Main.jsx

```diff
@@ -151,7 +149,7 @@ export default class Main extends Component {
         <ExtensionUI
           disablePreviewOpen={!!contentSyncUrl}
           contentSlug={contentSlug || initalValue}
-          previewUrl={this.getPreviewUrl()}
+          previewUrl={this.getPreviewUrl}
```

Comment:
Don't we still need to invoke `getPreviewUrl`, it looks like it's still a function

Reply:
Good catch, I thought that might be the issue initially. I'll update this now.